### DISCUSSION TITLE
Revise placement of 'Unfollow' on mobile width

### DISCUSSION THREAD

**Discussion by akutch** [deso-protocol/dips#47] [Medium CIPs]
## The Problem

![mobile](https://user-images.githubusercontent.com/6456722/125289420-35a40400-e2ed-11eb-81b2-84e57b5fd4b3.png)

## 💡 Proposed solution

On creator posts, we remove `Unfollow`, and only leave option to `Follow`

Folks will be able to `Unfollow` others directly from their profile. This behavior will allow us to avoid accidental `Unfollow` clicks, as this is more of a destructive action than `Follow` IMO.
 
⭐ As a bonus, I propose anywhere a Creator username is listed, we make that link to their profile. Starting with the 'thread' page title. 

### Rough mockup 
_Don't mind the spacing_

Following | Not following 
-|-
Unfollow link is removed | Follow link remains (blur for privacy/no hurt feelings)
![IMG_7E90C1CA29A7-1](https://user-images.githubusercontent.com/6456722/125290354-35583880-e2ee-11eb-8299-e6a62fe34c14.jpeg) | ![not-following](https://user-images.githubusercontent.com/6456722/125290337-312c1b00-e2ee-11eb-820b-5691156db44c.png)

## Supporting evidence 🕵️‍♀️ 
This causes a bit of user pain, as emphasized in [this post](https://bitclout.com/posts/61635a7cb243e016dbd1d456fc0b78da659374e717db187b3b775fd2b495b0c1): 

<img width="595" alt="Screen Shot 2021-07-13 at 7 15 12 AM" src="https://user-images.githubusercontent.com/6456722/125442714-330e1e98-0051-4c02-865b-d538e5bac66c.png">

**Comment by ItsAditya-xyz** [comment]
Agreed with making the thread tab a link to user's profile.

and as far as removal of unfollow button is concerned, i think if unfollow button is removed, the follow button should be removed as well otherwise the UI will look weird (imagine following a user from post and suddenly that button disappears).

or maybe we can wait till we have profile hover feature in frontend.

**Comment by tijno** [comment]
Haha great to see this change proposed.

I have had this exact issue since day 1.

![20210712-KHbBgR3F](https://user-images.githubusercontent.com/69529928/125354796-24461080-e35c-11eb-9017-b09a1c756ee1.png)

Re proposals:

1 - yes link thread title to profile, but include some icon or formatting to make it clear its a link to a profile?
2 - IG has the follow link next to the profile name, where bitclout has the coin price. It shows the unfollow link in the ... menu - this seems sensible. Maybe the follow link can go below the profile image? I dont think it should be moved to the `...` menu - following is too important an action.

**Comment by akutch** [comment]
Both great suggestions! 

1 - note the underline on the username in the thread title 
2 - pretend I **do not** follow maebeam in the "after" screenshot 
3 - not included in the mockup, but `Unfollow` will be added to the `...` menu

Before | After
-|-
<img width="310" alt="before" src="https://user-images.githubusercontent.com/6456722/125442265-2a9fce05-fc33-42ff-899b-4f799b032f5c.png"> | <img width="308" alt="after" src="https://user-images.githubusercontent.com/6456722/125442264-4d8c1628-3e21-463a-b35e-547ae4790cb3.png">

Thoughts on this?

**Reply by tijno** [reply]
yes i like this a lot - much better.

**Reply by akutch** [reply]
Great! I should be able to get to it this week. 

Should I wait until this is approved?

**Reply by akutch** [reply]
Still wondering about this @tijno 🙂

**Reply by tijno** [reply]
need  core dev approval 

hopefully @lazynina or @maebeam get a chance to review & approve.

you have community ok clearly.

**Reply by lazynina** [reply]
Thanks for the mocks @akutch - I have experienced this problem as well. However, putting the follow button under the avatar seems a bit out of place to me as it clashes with the post text in my opinion. I'm not a design expert though

@maebeam Any thoughts on how we could reposition the follow button to reduce errant follow/unfollow behavior on mobile? 

Some ideas that have been thrown around:
1. Feed post dropdown menu
2. Remove the follow button on mobile altogether
3. Bump the coin price and buy element to the next row instead of the follow button
4. Place below avatar as suggested by @akutch

**Reply by tijno** [reply]
Just a consideration - swap coin price and follow/unfollow. 

Price under name is more prominent and follow button to right of name is in line with user experience on IG where its to the right of the name too (with unfollow in the ... menu)

**Reply by maebeam** [reply]
I'd like to see a mock with coin price and follow swapped. Could look good

**Reply by akutch** [reply]
I’ll put that together this afternoon. Thanks!

**Reply by tijno** [reply]
@maebeam quick edit in browser to give you an idea.

![20210726-PMZxMWmz](https://user-images.githubusercontent.com/69529928/127019131-23a61c6e-9114-41dd-8b8f-dbec1051cc43.png)

**Reply by maebeam** [reply]
I was thinking coin price below the avatar
